sirepo/package_data/template/srw/parameters.py.jinja

```diff
@@ -502,6 +501,15 @@ def _rsopt_set_params({{ rsOptFuctionSignature() }}):
 
 {% endif %}
 
+def epilogue():
```

Comment:
i don't see the point of epilogue function if it's just two lines in a template for server side only. Just inline it. The srwMain could we in an if __name__ == main, I think, so the code could be simplified for rsOpt, too.